### ISSUE TITLE
external-lib-deps --missing doesn't find PPX dependencies

### ISSUE THREAD

**Issue by g2p** [ocaml/dune#1416]
Dependencies mentioned in a pps stanza still need to be installed manually

**Comment by emillon** [comment]
I tried to reproduce this but didn't manage to:

```dune
(executable
 (name xx)
 (preprocess (pps ppx_deriving.std ppx_tyre))
)
```

(I have installed `ppx_deriving` but not `ppx_tyre`)

```
 % dune exec ./xx.exe                       
File "dune", line 3, characters 35-43:
3 |  (preprocess (pps ppx_deriving.std ppx_tyre))
                                       ^^^^^^^^
Error: Library "ppx_tyre" not found.
Hint: try: dune external-lib-deps --missing ./xx.exe
 % dune external-lib-deps --missing ./xx.exe
Done: 0/0 (jobs: 0)Error: The following libraries are missing in the default context:
- ppx_tyre
Hint: try: opam install ppx_tyre
```

Do you have an example I could look at?

**Comment by diml** [comment]
This should be fixed by #1607
